### PR TITLE
[Restored] Bump Twilio Library to Version 5.x

### PR DESCRIPTION
**Pull request by acamino**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.3.0
```
Comment (well1791) [inline .ruby-version]: @ronualdo should we specify the version through this file?

File: .ruby-gemset [hunk 2]
```diff
@@ -0,0 +1 @@
+sms2fa-sinatra
```
Comment (well1791) [inline .ruby-gemset]: @ronualdo In order to keep this files here, I think we should give more context about the use of this in the readme.

File: .ruby-version (hunk 1 above)
Comment (ronualdo) [inline .ruby-version]: We got to the agreement that we should not add these files. I'll remove them

File: .ruby-gemset (hunk 2 above)
Comment (ronualdo) [inline .ruby-gemset]: We decided to remote these files from the repo.

**Pull request merged**